source/stylesheets/base/_navigation.scss

```diff
@@ -12,35 +12,52 @@
   .container {
     display: flex;
     flex-direction: row;
+    position: relative;
 
-    img {
-      transition: height 0.1s ease;
+    .blog-title {
+      color: $red;
```

Comment:
So, this is where utility classes can get tricky. Didn't you have a class in your markup, `.red`, doing this exact same thing?